docs-sphinx/_toc.yml

```diff
@@ -28,8 +28,6 @@ subtrees:
                         title: "Python objects"
                       - file: user-guide/how-to-convert-json
                         title: "JSON"
-                      - file: user-guide/how-to-convert-uproot
```

Comment:
This should be done in the uproot docs in my opinion.

Reply:
[It is](https://uproot.readthedocs.io/en/latest/basic.html#reading-a-tbranch-as-an-array), but we don't know where users will be coming from, which site they'll find first. We ought to at least link users to the Uproot documentation, to funnel them in from wherever they might be coming from to where the information is.

Reply:
Maybe this page should just a direct link to the uproot docs, them.